info.xml

```diff
@@ -24,14 +24,18 @@
   <comments>This module's origins are Xaviers Dutoit's (xavier@tttp.eu) and Bj&#xF6;rn Endres' (endres@systopia.de) Project60 SEPA branches.</comments>
   <classloader>
     <psr4 prefix="Civi\" path="Civi"/>
+    <psr0 prefix="CRM_" path="."/>
   </classloader>
   <civix>
     <namespace>CRM/Sepa</namespace>
-    <format>22.05.2</format>
+    <format>23.02.1</format>
   </civix>
   <mixins>
     <mixin>menu-xml@1.0.0</mixin>
     <mixin>mgd-php@1.0.0</mixin>
     <mixin>setting-php@1.0.0</mixin>
+    <mixin>smarty-v2@1.0.1</mixin>
```

Comment:
The update to 23.02.1 removes the mixin files from the extension. Which then would expect the functionality to be present in CiviCRM core. So again I am not sure whether that is the case with CiviCRM 5.45

Reply:
This is also being checked by *Civix* and the `info.xml` adjusted accordingly. E.g. the `menu-xml` mixin, this got added to CiviCRM core with version `5.45` (see https://github.com/civicrm/civicrm-core/commit/7eab943a07f6d2fbc78e0f5ff062b52415f19a52).

Reply:
ok cool.